homeassistant/components/overkiz/sensor.py

```diff
@@ -28,7 +32,17 @@
 from . import HomeAssistantOverkizData
 from .const import DOMAIN, IGNORED_OVERKIZ_DEVICES
 from .coordinator import OverkizDataUpdateCoordinator
-from .entity import OverkizDescriptiveEntity, OverkizEntity, OverkizSensorDescription
+from .entity import OverkizDescriptiveEntity, OverkizEntity
+
+
+@dataclass
+class OverkizSensorDescription(SensorEntityDescription):
+    """Class to describe an Overkiz sensor."""
+
+    native_value: Callable[
+        [str | int | float], str | int | float
+    ] | None = lambda val: val
```

Comment:
Can `native_value` ever be `None`?
You provide a default lambda, and I couldn't find anything below that sets it to `None`.

Reply:
Good catch! What would you think is better, provide the default lambda and remove None? Or just set it to None and add a lambda when a transformation is required.


https://github.com/home-assistant/core/blob/dc3f21dd1e353569d1a1ebaef481886b07430822/homeassistant/components/overkiz/sensor.py#L387-L388

Reply:
I think hasattr always returns None doesn't it?
I think I would have kept None, and make that the default.
Then only apply the transformation if it isnt None.

Reply:
>The hasattr() function returns True if the specified object has the specified attribute, otherwise False.

I will see if changing it to None will work with current code, otherwise I will add another check to see if native_value is None.

Reply:
```python
 if self.entity_description.native_value is None:
    return state.value
return self.entity_description.native_value(state.value)
```

Reply:
I just solved it in a similar way, thanks! :)